setup.cfg

```diff
@@ -27,8 +27,8 @@ classifiers =
 packages = find:
 python_requires = >=3.8
 install_requires =
-    aiidalab>=21.09.0
-    aiida-aurora>=0.3.2
+    aiidalab
+    aiida-aurora
```

Comment:
Any reason to remove the minimal versions? I would say if we have the information of what is the oldest version of a dependency with which the code worked correctly, it is not bad to keep that info in the code (we may need to update this since some changes would depend on the upcoming release of `aiida-aurora`, for example)

Reply:
I think I did this because there was a collision with aiida-aurora 0.3.2, but there shouldn't be any more. I'll remove it.